lib/stdlib/src/proplists.erl

```diff
@@ -39,6 +39,7 @@
 %% etc.</p>
 %%
 %% % @type property() = atom() | tuple()
+%% % @type proplist() = [property()]
```

Comment:
A question to the reviewers: since these types are defined (not opaque) and exported, do we really need to repeat their declaration? Can't Edoc find that out by itself?